fleet/lib/rook-ceph-conf/charts/kueyen/templates/obc-loki.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: logs-chunks-storage
```

Comment:
1. Filename doesnt match the name of the CRD.
2. Bucket Name doesnt match the name of the CRD.
3. you sure you want to create a bucket callend "logs-chunks-randomstuff" (check Documentation on `generateBucketName`)
4. several OBC need different files. (as the filename is $kind-$name)
5. no quota on the bucket.